### ISSUE TITLE
Error message in C2

### ISSUE THREAD

**Issue by Robinlovelace** [Robinlovelace/geocompr#696]
Split out from #691, we're currently seeing the following error: 

```
 Quitting from lines 694-695 (02-spatial-data.Rmd) 
Error: $ operator is invalid for atomic vectors
```

E.g.: here https://github.com/Robinlovelace/geocompr/runs/4566055217?check_suite_focus=true#step:4:918

Plan: comment out the offending code for now as it's only one line, reinstate when it works again due to upstream fixes. May be worth raising an issue in upstream libs.

**Comment by Robinlovelace** [comment]
Same issue appears in C3: 

```
 Quitting from lines 664-665 (03-attribute-operations.Rmd) 
Error: $ operator is invalid for atomic vectors
```

https://github.com/Robinlovelace/geocompr/runs/4566152753?check_suite_focus=true#step:4:1893

**Comment by Robinlovelace** [comment]
We can serve static versions of the figures as a quick fix. In c2:

![image](https://user-images.githubusercontent.com/1825120/146617327-45919232-a6a3-4d9d-a158-afa87f47381b.png)

And C3:

![image](https://user-images.githubusercontent.com/1825120/146617366-7308535b-30f6-4c87-83f7-21702c7d993b.png)

**Comment by Robinlovelace** [comment]
Closed by #d1aabd6ae427f858bacd1b2bea065ec42b17056e
